Codegen/src/Nodes/TypedDataNode.cs

```diff
@@ -986,7 +987,7 @@ public static List<ITextLine> InsertChildren(ColumnsLayout? layout, List<string>
                 {
                     if (child.IsFlagSet(Flag.IsTypedefCopyNode))
                     {
-                        lines.AddRange(child.Lines);
+                        lines.AddRange((child as LinearNodeSourceCodeMapper.LinearCopyNode).Lines);
```

Comment:
No need to cast here, polymorphism for the win !

Reply:
It's right; done.
https://github.com/TypeCobolTeam/TypeCobol/pull/1648/commits/e087d9212c17eba494fe8cca891e34783e2e4570